### PR TITLE
Multisearch

### PR DESCRIPTION
**Pull request by mmacata**
This PR adds a multisearch.
Necessary for it is the basigx multisearch (https://github.com/terrestris/BasiGX/pull/111)
and the momo3-backend adjustments (https://github.com/terrestris/momo3-backend/pull/44)

### CODE REVIEW

**Comment by annarieger** [comment]
Nice one! :+1: 

Just one small remark.. Please add license and copyright informations (can be copied from some another momo component class) as well as small description for each JS class :angel:

File: classic/sass/src/view/form/field/MultiSearchCombo.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    display: none;
+}
+
+.multisearch-refresh-trigger:before {
+    content: '\f021';
+}
+
+.multisearch-settings-trigger:before {
+    content: '\f013';
+}
```
Comment (marcjansen) [inline classic/sass/src/view/form/field/MultiSearchCombo.scss]: Nit-pick: Newline missing

File: classic/src/view/container/MultiSearchSettings.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Ext.define('MoMo.client.view.container.MultiSearchSettings',{
```
Comment (marcjansen) [inline classic/src/view/container/MultiSearchSettings.js:25]: File header missing.

File: classic/src/view/form/field/MultiSearchCombo.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        searchLayerBlackList: [
+            "hoverVectorLayer",
+            "OSM-WMS GRAY",
+            // following is a grouplayer. If it should be searchable,
+            // sublayers need to be used
+            "exploitable_groundwater"
+        ]
+    }
+
+});
```
Comment (marcjansen) [inline classic/src/view/form/field/MultiSearchCombo.js]: Nit-pick: Newline missing

File: classic/src/view/grid/MultiSearchGazetteerGrid.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+    viewModel: {
+        data: {
+            /*i18n*/
+            title: ''
+            /*i18n*/
+        }
+    }
+
+});
```
Comment (marcjansen) [inline classic/src/view/grid/MultiSearchGazetteerGrid.js]: Nit-pick: Newline missing

File: classic/src/view/grid/MultiSearchWFSSearchGrid.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    viewModel: {
+        data: {
+            /*i18n*/
+            title: '',
+            hideToolTooltip: ''
+            /*i18n*/
+        }
+    }
+
+});
```
Comment (marcjansen) [inline classic/src/view/grid/MultiSearchWFSSearchGrid.js]: Nit-pick: Newline missing

File: classic/src/view/form/field/MultiSearchCombo.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
+Ext.define("MoMo.client.view.form.field.MultiSearchCombo", {
```
Comment (marcjansen) [inline classic/src/view/form/field/MultiSearchCombo.js:25]: File header missing.

File: classic/src/view/grid/MultiSearchGazetteerGrid.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+Ext.define("MoMo.client.view.grid.MultiSearchGazetteerGrid",{
```
Comment (marcjansen) [inline classic/src/view/grid/MultiSearchGazetteerGrid.js:23]: File header missing.

File: classic/src/view/grid/MultiSearchWFSSearchGrid.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
+Ext.define("MoMo.client.view.grid.ObjectSearchGrid",{
```
Comment (marcjansen) [inline classic/src/view/grid/MultiSearchWFSSearchGrid.js:23]: File header missing.

**Review by marcjansen — APPROVED** [review]
Only minors, please merge once addressed. Thanks!

**Comment by mmacata** [comment]
All comments adressed now

**Comment by marcjansen** [comment]
build failure is probably related to phantomjs/slimerjs no being dowloadable right now.

**Comment by marcjansen** [comment]
Hmmm. probably not related to any of the above? Can we try to run `npm i && npm test` in a fresh clone (no `node_modules`)?

**Comment by mmacata** [comment]
first build failure was just a trailing white space ;)

**Comment by weskamm** [comment]
merging now

**Pull request merged**